south-african-journal-of-animal-science.csl

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
   <info>
-    <title>South African Journal of Animal Science</title>
+    <title>south-african-journal-of-animal-science.csl</title>
     <title-short>South African Journal of Animal Science</title-short>
-    <id>http://csl.mendeley.com/styles/21406231/South-African-Journal-of-Animal-Science</id>
-    <link href="http://www.zotero.org/styles/poultry-science" rel="self"/>
-    <link href="http://www.zotero.org/styles/asa-cssa-sssa" rel="template"/>
```

Comment:
We would still like to have a "template" link. It looks like that should be `<link href="http://www.zotero.org/styles/poultry-science" rel="template"/>`, if you indeed based this style off of poultry-science.csl.